id-repository/id-repository-core/src/test/java/io/mosip/idrepository/core/test/builder/RestRequestBuilderTest.java

```diff
@@ -179,12 +210,17 @@ public void testBuildRequestEmptyTimeout() throws IdRepoDataValidationException
 		MockEnvironment environment = new MockEnvironment();
 		environment.merge(env);
 		environment.setProperty("mosip.idrepo.audit.rest.timeout", "");
-
+		mapBuilder.get("mosip.idrepo.audit").put(".rest.timeout", "");
+		ReflectionTestUtils.setField(restBuilder, "mapBuilder", mapBuilder);
 		ReflectionTestUtils.setField(restBuilder, "env", environment);
-
-		restBuilder.buildRequest(RestServicesConstants.AUDIT_MANAGER_SERVICE, auditBuilder
-				.buildRequest(AuditModules.ID_REPO_CORE_SERVICE, AuditEvents.CREATE_IDENTITY_REQUEST_RESPONSE, "id",IdType.ID, "desc"),
-				AuditResponseDTO.class);
+		try {
+			restBuilder.buildRequest(RestServicesConstants.AUDIT_MANAGER_SERVICE, auditBuilder
+					.buildRequest(AuditModules.ID_REPO_CORE_SERVICE, AuditEvents.CREATE_IDENTITY_REQUEST_RESPONSE, "id",IdType.ID, "desc"),
+					AuditResponseDTO.class);
+		}
+		catch(Exception e) {
+				e.printStackTrace();

```

Comment:
Don't catch common Exception, catch specific exception if any. In tests you can throw them, no need to catch.